print_docs.py

```diff
@@ -666,7 +666,9 @@ def copy_static_files(path):
     shutil.copy(filename, path)
 
 def write_decl_txt(loc_map):
-  with open_outfile('decl.txt') as out:
+  # it's not a bitmap, but apparently this tricks Github Pages / nginx into gzipping it
```

Comment:
I'm not sure whether I should be disgusted or impressed.  Either way, if it works let's merge it.

Reply:
Would `.js` leave you less disgusted?

Reply:
```suggestion
  # it's not a javascript file, but apparently this tricks Github Pages / nginx into gzipping it
```